uint64 timestamp					# time since system start (microseconds)

float32[3] force
float32    thrust_coff              # thrust_coff = hover_thrust / uav_mass / 9.81